src/sdk/base/mediastream-factory.js

```diff
@@ -181,7 +181,11 @@ export class MediaStreamFactory {
         };
       }
     } else {
-      mediaConstraints.audio = constraints.audio;
+      if (constraints.audio.source === MediaFormatModule.AudioSourceInfo.SCREENCAST) {
+        mediaConstraints.audio = true;
```

Comment:
Please respect user's input. Why it cannot be false?

Reply:
`constraints.audio.source === MediaFormatModule.AudioSourceInfo.SCREENCAST` means user is requesting audio. Resolve this one.

Reply:
I think when the audio source set to `screen-cast`, that is equal to `true`